components/script/script_thread.rs

```diff
@@ -1280,6 +1280,14 @@ impl ScriptThread {
             if document.is_fully_active() {
                 window.reflow(ReflowGoal::UpdateTheRendering, can_gc);
             }
+            // Update the rendering of those does not require a reflow.
+            // e.g. animated images.
+            if document
+                .image_animation_manager()
+                .image_animations_present()
+            {
+                document.update_animated_image_active_frames();
+            }
```

Comment:
I suspect that the image animation manager should use interior mutability, but for now, please do something like this:

```suggestion
            if document
                .image_animation_manager()
                .image_animations_present()
            {
                 self
                      .image_animation_manager_mut()
                     .update_active_frames(self.window(), self.current_animation_timeline_value())
            }
```

In addition, I think this should likely happen *before* reflow. It's a lot like updating canvases. When the image frame updates, it could trigger a reflow somehow, perhaps by forcing a repaint of nodes that use the animated image.